### PR TITLE
chore(perf): add sample scenario and docs

### PR DESCRIPTION
**Pull request by majorgreys**
## Description

Sample scenario with documentation.

## Checklist
- [ ] Added to the correct milestone.
- [ ] Tests provided or description of manual testing performed is included in the code or PR.
- [ ] Library documentation is updated.
- [ ] [Corp site](https://github.com/DataDog/documentation/) documentation is updated (link to the PR).

### CODE REVIEW

**Review by Kyle-Verhoog — APPROVED** [review]

File: benchmarks/base.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  # cleaning up unused files
+  && apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false \
+  && rm -rf /var/lib/apt/lists/*
+
+# sirun
+RUN curl -sL https://github.com/DataDog/sirun/releases/download/v0.1.8/sirun-v0.1.8-x86_64-unknown-linux-gnu.tar.gz | tar zxf - -C /tmp && mv /tmp/sirun /usr/bin
+
+COPY ./common/entrypoint /app/
+COPY ./common/benchmark /app/
+COPY ./common/meta.yaml /app/
```
Comment (Kyle-Verhoog) [inline benchmarks/base.Dockerfile:26]: Doesn't look like this file is checked-in

```
 => ERROR [ 7/10] COPY ./common/meta.yaml /app/                                                                                                              0.0s
------
 > [ 7/10] COPY ./common/meta.yaml /app/:
------
failed to compute cache key: "/common/meta.yaml" not found: not found
```

File: benchmarks/README.rst [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+===================
+
+New scenario
+------------
+
+Copy ``sample`` directory and edit the ``meta.yaml`` file to specify the executable. Add any additional requirements to the ``requirements.txt``.
+
+Build image::
+
+  docker build -t benchmark-<scenario> --build-arg <scenario> -f base.Dockerfile .
```
Comment (Kyle-Verhoog) [inline benchmarks/README.rst:11]:
```suggestion
  docker build -t benchmark-<scenario> --build-arg BENCHMARK=<scenario> -f base.Dockerfile .
```

**Review by Kyle-Verhoog — CHANGES_REQUESTED** [review]

File: benchmarks/base.Dockerfile (hunk 1 above)
Comment (Yun-Kim) [reply benchmarks/base.Dockerfile:26]: Should this be `sample/meta.yaml`?

File: benchmarks/base.Dockerfile [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+RUN python3 -m venv $VIRTUAL_ENV
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+
+RUN pip install -r requirements.txt
+
+# For performance testing
+ENV DDTRACE_GIT_COMMIT_ID ""
+ENV PIP_INSTALL_WHEELS ""
+ENV SIRUN_NO_STDIO 0
+ENV K6_STATSD_ENABLE_TAGS "true"
```
Comment (P403n1x87) [inline benchmarks/base.Dockerfile:42]: Should this k6-specific line be removed from the sample?

**Review by Kyle-Verhoog — APPROVED** [review]

**Review by Yun-Kim — APPROVED** [review]

**Pull request merged**